filesystems/xfs/xfstests/README.md

```diff
@@ -0,0 +1,22 @@
+# xfstests test suite
+Wrapper for xfstests test suite, see http://git.kernel.org/?p=fs/xfs/xfstests-dev.git;a=summary \
+Test Maintainer: [Xiong Murphy Zhou](mailto:xzhou@redhat.com)
+
+## How to run it
+
+### Configure which test cases to run, e.g.
+```bash
+$ export TEST_PARAM_FSTYPE=xfs
+$ export TEST_PARAM_RUNTESTS=generic/001
+```
+
+### Dependencies
+Dependencies will be automatically installed with dep-install.sh when you execute \
+'make run', however to see a complete list of dependencies, please see \
```

Comment:
Nitpick: I think it's gonna sound more natural if you write "however, for a complete list of dependencies, please see".

Reply:
I cleaned it up based on your suggestion.